libs/auth-nest-tools/src/lib/auth.middleware.ts

```diff
@@ -56,6 +57,9 @@ export class AuthMiddleware implements Middleware {
 
     context.init.headers = Object.assign({}, context.init.headers, {
       authorization: bearerToken,
+      ...(this.options.authID && {
+        ['Authorization-Identity']: bearerToken,
+      }),
```

Comment:
This looks very non-standard. Where does this come from?

Would be good to document. If this is needed in a single integration, we should just do it in another middleware, or add a more flexible option to this middleware: `headerName = 'authorization'`.

Reply:
I believe this is a temporary solution from the providers of the Fasteignir API and they will update it eventually to require something else than the same bearer token as the Authorization.
But for now they requested it to be the same.
I'm not sure if this is just a one off for `Fasteignir-v1` or if they will require this for all `SKRA-Protected` in the coming future.

Reply:
Ok.

Then can we move this code somewhere else? :)

Reply:
cc @vigfusg @saevarma. I always get worried when I see non-standard security logic. 😅

Should we ask for more information about what they need this non-standard header for? Why the access token is not enough?